diff --git a/node_modules/react-native-audio-api/src/specs/NativeAudioAPIModule.ts b/node_modules/react-native-audio-api/src/specs/NativeAudioAPIModule.ts
--- a/node_modules/react-native-audio-api/src/specs/NativeAudioAPIModule.ts
+++ b/node_modules/react-native-audio-api/src/specs/NativeAudioAPIModule.ts
@@ -3,7 +3,7 @@
 import { TurboModuleRegistry } from 'react-native';
 import { AudioDevicesInfo, PermissionStatus } from '../system/types';

-type OptionsMap = { [key: string]: string | boolean | number | undefined };
+type OptionsMap = { [key: string]: string };
 type NotificationOpResponse = { success: boolean; error?: string };
 type NotificationType = 'playback' | 'recording' | 'simple';
 type AudioFocusType =
